### PR TITLE
chore: update dependencies, module-base to 1.4.1

### PR DESCRIPTION
**Pull request by fmalcher**
tested with the latest experimental Companion release 3.99.0

This contains no breaking changes, just package updates. So probably this will become v3.0.1, right?
I haven't updated manifest and package.json yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,16 +34,18 @@
 		"node": "^18.12"
 	},
 	"dependencies": {
-		"@companion-module/base": "~1.0",
-		"rxjs": "~7.5.0",
-		"soundcraft-ui-connection": "1.4.0"
+		"@companion-module/base": "^1.4.1",
+		"bufferutil": "^4.0.7",
```
Comment (Julusian) [inline package.json:38]: do you need/use bufferutil and utf-8-validate?
From memory they are both native modules and dont get packaged up properly by our build process

**Comment by Julusian** [comment]
Yeah it could be v3.0.1 or v3.1.0.
I would probably go with v3.1.0 as it is a major version bump of the connection library, but v3.0.1 is also fine

File: package.json (hunk 1 above)
Comment (fmalcher) [reply package.json:38]: I changed the build process for the connection lib. It works in my own Node experiments. However, the conpanion module build complained about those two so I added them here.

Comment (Julusian) [reply package.json:38]: ah right. they are optional dependencies of `ws`, so webpack will warn about them. 
either way is fine really

**Pull request merged**